test/docker/docker_utilities.go

```
@@ -29,18 +31,23 @@ func AwaitDatabaseUp(t *testing.T, composeFile string, adminContainerName string
 		"--timeout", "120")
 }
 
-func GetDatabaseListings(t *testing.T, composeFile string, influxContainerName string) []string {
-	raw := executeCommandInContainer(t, composeFile, influxContainerName, "influx", "bucket", "list", "--json")
+func GetDatabaseListings(t *testing.T, composeFile string, influxContainerName string) ([]string, error) {
+	jsonData := executeCommandInContainer(t, composeFile, influxContainerName, "influx", "bucket", "list", "--json")
 	var bucketList []map[string]interface{}
 	var listing []string
-	jsonData := raw
 	err := json.Unmarshal([]byte(jsonData), &bucketList)
 	if err != nil {
 		fmt.Println("Error while decoding data ", err.Error())
```

No need to print the error. Just let it propagate and let the caller handle it.